scripts/nz/README.md

```diff
@@ -39,7 +39,7 @@ apt-get install docker.io.
 Getting Data
 ----
 
-1.  Start by [finding the Street Address (Electoral) dataset](https://data.linz.govt.nz/search/?q=street+address+electoral)
+1.  Browse to the [NZ Street Address dataset](https://data.linz.govt.nz/layer/3353-nz-street-address/)
     on [LINZ Data Service](https://data.linz.govt.nz/), and select “download” from
     the drop-down menu:
```

Comment:
We should update the screenshots to match the new functionality.
![image](https://cloud.githubusercontent.com/assets/13267/20643976/c1135846-b48b-11e6-8f35-fbe08305770f.png)

I also see a significantly different size (360MB).